### PR TITLE
[HSEARCH] 7.2.0.Alpha1

### PR DESCRIPTION
**Pull request by marko-bekhta**
The series summary is the same as in the post https://github.com/hibernate/in.relation.to/pull/193

### CODE REVIEW

File: _data/projects/search/releases/7.2/series.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      summary: Amazon IAM authentication for Elasticsearch/OpenSearch
+    - artifact_id: hibernate-search-mapper-orm-jakarta-batch-core
+      summary: Jakarta Batch mass indexing job for the Hibernate ORM mapper - Core
+    - artifact_id: hibernate-search-mapper-orm-jakarta-batch-jberet
+      summary: Jakarta Batch mass indexing job for the Hibernate ORM mapper - JBeret specifics
+    - artifact_id: hibernate-search-v5migrationhelper-orm
+      summary: Helper for migrating from Hibernate Search 5 to Hibernate Search 6/7 (Hibernate ORM mapper + Lucene backend)
+integration_constraints:
+  java:
+    version: 11, 17 or 21
```
Comment (yrodiere) [inline _data/projects/search/releases/7.2/series.yml]: Should we mention 22? :thinking:

File: search/releases/7.2/index.adoc [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+:awestruct-project: search
+:awestruct-series_version: "7.2"
+:page-interpolate: true
+:latest-release-version: #{series(page, page.series_version).releases.first.version}
+:hsearch-doc-url-prefix: #{reference_doc(site.projects[page.project], series(page, page.series_version)).html_url}
+
+=== Dependency upgrades
+
+[[orm-version]]
+Hibernate ORM (link:{hsearch-jira-url-prefix}/HSEARCH-5150[HSEARCH-5150])::
```
Comment (yrodiere) [inline search/releases/7.2/index.adoc]: I would recommend dropping the references to Jira issues here. They will likely be painful to maintain as you release new alphas/betas, and don't bring much in this particular case -- as opposed to the blog post which is specifically about the new alpha.

File: search/releases/7.2/index.adoc [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+        .fetchHits( 20 );
+----
+<1> Start creating the `.withParameters()` predicate.
+<2> Access the query parameter `title-param` of `String` type when constructing the predicate.
+<3> Access the query parameter `genre-param` of `Genre` enum type when constructing the predicate.
+<4> Set parameters required by the predicate at the query level.
+====
+
+[[distance-projection]]
+=== @DistanceProjection to map a constructor parameter to a distance projection
```
Comment (yrodiere) [inline search/releases/7.2/index.adoc]:
```suggestion
=== `@DistanceProjection` to map a constructor parameter to a distance projection
```

**Review by yrodiere — APPROVED** [review]

File: _data/projects/search/releases/7.2/series.yml (hunk 1 above)
Comment (marko-bekhta) [reply _data/projects/search/releases/7.2/series.yml]: IDK 🤷🏻‍♂️ 😨 🤔 😃 
Looking at the ORM -- no mention of 22 there 🙈 
soooo I'll add it if you think we should 😃

**Pull request merged**